test/integration.test.js

```diff
@@ -48,4 +51,27 @@ describe('Integration testing generateFromFile() to make sure the result of the
     const file = await readFile(path.join(outputDir, 'index.html'), 'utf8');
     expect(file).toMatchSnapshot();
   });
+
+  it('should ignore specified files with noOverwriteGlobs', async () => {
+    const outputDir = generateFolderName();
+    // Manually create a file to test if it's not overwritten
+    if (!await exists(outputDir)) {
+      mkdirSync(outputDir, { recursive: true });
+    }
+    // eslint-disable-next-line sonarjs/no-duplicate-string
+    const testFilePath = path.join(outputDir, 'index.html');
+    await writeFile(testFilePath, '<script>const initialContent = "This should not change";</script>');
+
+    // based on the html-template documentation, the default output is index.html
+    const generator = new Generator('@asyncapi/html-template@0.28.0', outputDir, {
+      forceWrite: true,
```

Comment:
if you also pass `debug: "true` here then in this test we can also validate that proper debug info is shown to the user

```
expect(console.log).toHaveBeenCalledWith(`Skipping overwrite for ${testFilePath}`);
```

yeah, something like that should work but you of course need to check, I just provide pseudo code

Reply:
I think you missed this comment

Reply:
is ``Skipping overwrite for ${testFilePath}`` the exact message that will be displayed when we add `debug:true` as an argument to the generator?